### PR TITLE
Add new Releases section

### LINKED ISSUE #1258: Create "Releasing" section under somewhere TBD
We could use a page like this, but then in the general Plone docs in a section about contributing, not specific to `plone.api`.

_Originally posted by @mauritsvanrees in https://github.com/plone/plone.api/issues/476#issuecomment-1136828066_

### PR DESCRIPTION
**Pull request by stevepiercy**
Closes #1258.

@mauritsvanrees

### CODE REVIEW

**Review by mauritsvanrees — COMMENTED** [review]
This is fine, except that I don't think Volto is using zest.releaser.
@sneridagh Can you write a sentence about that?

File: docs/contributing/index.md [hunk 1]
```diff
@@ -139,6 +140,15 @@ Volto
     See {doc}`../volto/contributing/index`.
 
 
+(contributing-releases-label)=
+
+## Releases
+
+The Plone [Release Team](https://plone.org/community/teams/release-team) manages the release of new versions of Plone.
+
+We use [`zest.releaser`](https://zestreleaser.readthedocs.io/en/latest/) for releasing the packages used in Plone, including [Plone core (`Products.CMFPlone`)](https://github.com/plone/Products.CMFPlone/), {doc}`Volto </volto/index>`, {doc}`/classic-ui/index`, {doc}`/plone.restapi/docs/source/index`, {doc}`/plone.api/index`, and {doc}`/backend/index`.
```
Comment (davisagli) [inline docs/contributing/index.md]:
```suggestion
We use [`zest.releaser`](https://zestreleaser.readthedocs.io/en/latest/) for releasing the Python packages used in Plone, including [Plone core (`Products.CMFPlone`)](https://github.com/plone/Products.CMFPlone/), {doc}`/classic-ui/index`, {doc}`/plone.restapi/docs/source/index`, {doc}`/plone.api/index`, and {doc}`/backend/index`.

We use [`release-it`](https://github.com/release-it/release-it) for releasing the Node.js packages used in Plone, including {doc}`Volto </volto/index>` and the [Classic UI mockup](https://github.com/plone/mockup).
```

**Comment by stevepiercy** [comment]
@sneridagh you use `release-it` for Volto, correct? If so, it's a quick fix I can do here.

**Review by davisagli — COMMENTED** [review]
@stevepiercy oops, I made this suggestion earlier but forgot to submit it

**Review by davisagli — APPROVED** [review]

**Pull request merged**